connectivity/check/deployment.go

```diff
@@ -266,7 +266,7 @@ func newDaemonSet(p daemonSetParameters) *appsv1.DaemonSet {
 							ReadinessProbe:  p.ReadinessProbe,
 							SecurityContext: &corev1.SecurityContext{
 								Capabilities: &corev1.Capabilities{
-									Add: []corev1.Capability{"NET_RAW"},
+									Add: []corev1.Capability{"NET_ADMIN", "NET_RAW"},
```

Comment:
Is this proposing to grant `NET_ADMIN` rights to connectivity test deployments in users' production environments? That seems like a sort of security footprint we are explicitly trying to avoid in a "safe" tool like this. :thinking:

Reply:
Good concern, and I overlooked that this granting applies to all deployments (not only to those running in host netns). Anyway, `NET_ADMIN` is required only for host netns pods which runs on special testing nodes which do not run Cilium (`--nodes-without-cilium`) for running tests from outside a cluster (e.g., to test N/S loadbalancing). I'm going to change the granting in this PR.

Reply:
LGTM thanks.